code/game/sound.dm

```diff
@@ -95,6 +95,13 @@ var/list/climb_sound = list(
 	'sound/effects/ladder4.ogg'
 )
 
+var/list/weld_sound = list(
+	'sound/items/welding1.ogg',
```

Comment:
More variety is good but there's nothing wrong with the old welder sound, add it to this list

Reply:
Have you listened to the sounds? They are very different from the old sound, it would seem out of place.

Reply:
Yes i have, I disagree. Requested change stands. please add it back in that list